libgalois/src/GraphTopology.cpp

```diff
@@ -539,6 +725,19 @@ katana::PGViewCache::BuildOrGetEdgeTypeAwareTopo(
   }
 }
 
+katana::ProjectedShuffleTopology*
```

Comment:
Should this be encapsulated in a shared pointer?

Reply:
I just followed a similar convention as was done for other classes in this file.